src/main/org/apache/tools/ant/taskdefs/optional/ssh/ScpToMessage.java

```diff
@@ -36,7 +36,7 @@
 public class ScpToMessage extends AbstractSshMessage {
 
     private static final int HUNDRED_KILOBYTES = 102400;
-    private static final int BUFFER_SIZE = 1024;
+    private static final int BUFFER_SIZE = 100*1024;
```

Comment:
Yes it's possible, but I didn't want to tie them together. The HUNDRED_KILOBYTES constant is used as a progress display threshold and IMHO should be named PROGRESS_DISPLAY_THRESHOLD or something like that, but that is left for another patch ;-)  I would then increase the value of PROGRESS_DISPLAY_THRESHOLD to at least 1MB, on local network 100KB file is quite small and progress display is really unnecessary.